scripts/bootstrap.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env sh
```

Comment:
`#!/usr/bin/env bash` or `#!/bin/sh` would make more sense to me. If `/usr/bin/env`  is a thing, then `/bin/sh` should also be a thing. Or are there systems where `env` is in the standard location, but `sh` isn't?

Reply:
> If /usr/bin/env is a thing, then /bin/sh should also be a thing.

As far as I understood it this definitely isn't the case. However, I don't personally know of an example. I just had it drilled in that /usr/bin/env sh or /usr/bin/env bash was the safe cross platform way to go.

Reply:
(although I am maybe 0.7 sure that the nix-one was copied from the rust one...)

Reply:
I've swapped it to /bin/sh